### ISSUE TITLE
[It broke]: V2 Update Download Location faces error

### ISSUE THREAD

**Issue by db-2001** [C9Glax/tranga#248]
### What is broken?

Testing the settings, the download location did not exist before hand yet Tranga was unable to move files

### Log-output

```C#
[9/14/2024 8:51:00 PM.562]        Server | System.Net.HttpListenerException (1229): An operation was attempted on a nonexistent network connection.
   at System.Net.HttpResponseStream.DisposeCore()
   at System.Net.HttpResponseStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at Tranga.Server.Server.SendResponse(HttpStatusCode statusCode, HttpListenerResponse response, Object content) in D:\GitHub\tranga-website\tranga\Tranga\Server\Server.cs:line 223
[9/14/2024 8:51:00 PM.583]        Server | System.Net.HttpListenerException (1229): An operation was attempted on a nonexistent network connection.
   at System.Net.HttpResponseStream.DisposeCore()
   at System.Net.HttpResponseStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at Tranga.Server.Server.SendResponse(HttpStatusCode statusCode, HttpListenerResponse response, Object content) in D:\GitHub\tranga-website\tranga\Tranga\Server\Server.cs:line 223
[9/14/2024 8:51:00 PM.583]        Server | System.Net.HttpListenerException (1229): An operation was attempted on a nonexistent network connection.
   at System.Net.HttpResponseStream.DisposeCore()
   at System.Net.HttpResponseStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at Tranga.Server.Server.SendResponse(HttpStatusCode statusCode, HttpListenerResponse response, Object content) in D:\GitHub\tranga-website\tranga\Tranga\Server\Server.cs:line 223
```

### Additional stuff

![image](https://github.com/user-attachments/assets/595d0740-c95d-4693-a235-a4c8425efc53)

**Comment by C9Glax** [comment]
Log is a different issue that I patched here 9b8b80cd2462ff0979c9405f1811c461a57f899f

**Comment by C9Glax** [comment]
Gotta still merge into Server-V2, gonna do that with the fix for the second issue
